### PR TITLE
Remove use of gemset variable

### PR DESCRIPTION
**Pull request by zjhuntin**

### CODE REVIEW

**Review by evgeni — APPROVED** [review]

**Review by evgeni — CHANGES_REQUESTED** [review]
wha, me--

but this needs to be "unique"

**Comment by zjhuntin** [comment]
Ahh, my bad

File: theforeman.org/scripts/test/test_develop.sh [hunk 1]
```diff
@@ -32,11 +32,11 @@ bundle exec rake rubocop
 
 # Database environment
 (
```
Comment (evgeni) [inline theforeman.org/scripts/test/test_develop.sh:34]: can you replace `gemset` with something like `jobid` and do before this block
```
jobid=$(echo ${JOB_NAME} | cut -d/ -f1)-${EXECUTOR_NUMBER}
```

the databases need to be unique

**Comment by evgeni** [comment]
> Ahh, my bad

nope, that's on me via 93ce4d5b01b11fe8e89151e4342e0cbbdd459f51

**Review by evgeni — APPROVED** [review]

**Pull request merged**